### PR TITLE
Synchronize configuration with URL

### PR DESCRIPTION
**Pull request by wasimpathan5**

### CODE REVIEW

File: demo.js [hunk 1]
```diff
@@ -105,7 +105,30 @@ function cellDialog(element, event) {
 	}});
 }
```
Comment (MikeVaz) [inline demo.js]: Our first pull request! :1st_place_medal:

File: demo.js [hunk 2]
```diff
@@ -105,7 +105,30 @@ function cellDialog(element, event) {
...
 
+/**
+ * Try to fetch the config from the URL query string.
+ */
+function fetchConfig() {
+	var searchStr = window.location.search;
+	var query = searchStr.split("=")[1];
+	try {
+		return JSON.parse(window.atob(query));
```
Comment (MikeVaz) [inline demo.js]: I think we would also check if there are any security vulnerabilities in such approach.

Comment (MikeVaz) [reply demo.js]: What is our length limits for a query string? Can we make it use friendly not encode everything?

Comment (MikeVaz) [reply demo.js]: Unit testing would be required for functional cases. DoD

Comment (wasimpathan5) [reply demo.js]: Regarding security - I think we are dealing in a correct way with the data.

Regarding length - I tried with a config with 50 cells data, it generated an encoded string of length 3876 and it worked fine. And, as a part of URL search string, I don't think there any theoretical limit on that. That depends on server, it must be able to handle the URI of unbounded length.

Unit testing - Sure, I will add that.

**Pull request merged**